src/go/k8s/api/redpanda/v1alpha2/redpanda_conversion.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+package v1alpha2
+
+import ctrl "sigs.k8s.io/controller-runtime"
+
+func (*Redpanda) Hub() {}
+
+// SetupWebhookWithManager will setup the manager to manage the webhooks
+func (r *Redpanda) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
I know the setup with manager is a common convention but it feels very strange to have this implemented on the CRD itself. Is doing so a common conversion for conversion webhooks?

Reply:
https://book.kubebuilder.io/multiversion-tutorial/webhooks#webhook-setup